### ISSUE TITLE
Unlocking lessons: newly added lessons aren't accessible to the user, admin accessibility

### ISSUE THREAD

**Issue by nscozzaro** [studyhub-co/physics-is-beautiful#16]
This is a bit of a corner-case, but if a user has finished all lessons in a module, then we add a new lesson, it should be accessible to the user. Currently it requires repeating the lesson before it to "trigger" the unlocking. 

Also, for testing or demonstration purposes the admin should have the ability to have all of the lessons unlocked.

**Comment by Nizebulous** [comment]
Pushed a fix for this.
